activerecord/lib/active_record/tasks/postgresql_database_tasks.rb

```diff
@@ -80,7 +80,11 @@ def structure_dump(filename, extra_flags)
 
       def structure_load(filename, extra_flags)
         set_psql_env
-        args = ["-v", ON_ERROR_STOP_1, "-q", "-f", filename]
+        args = if ENV["IGNORE_PG_LOAD_ERRORS"]
```

Comment:
Like @matthewd said we don't use environment variables to change behavior. What about using the already existent configuration `structure_load_flags`
 https://github.com/rails/rails/blob/fe343dcae003022e4ae9d507e4e22afabf498b58/activerecord/lib/active_record/tasks/database_tasks.rb#L44?

As the default is `nil` we could check the value of those flags and only add `ON_ERROR_STOP_1` when it is `nil`. If that config is anything but `nil` we use the config.